import-export-cli/utils/constants.go

```diff
@@ -57,11 +57,13 @@ const EnvKeysAllFileName = "env_keys_all.yaml"
 const MainConfigFileName = "main_config.yaml"
 const SampleMainConfigFileName = "main_config.yaml.sample"
 const DefaultAPISpecFileName = "default_api.yaml"
+const DefaultAWSDocFileName = "document.yaml"
 
 var LocalCredentialsDirectoryPath = filepath.Join(HomeDirectory, LocalCredentialsDirectoryName)
 var EnvKeysAllFilePath = filepath.Join(LocalCredentialsDirectoryPath, EnvKeysAllFileName)
 var MainConfigFilePath = filepath.Join(ConfigDirPath, MainConfigFileName)
 var SampleMainConfigFilePath = filepath.Join(ConfigDirPath, SampleMainConfigFileName)
+//var DefaultAPISpecFilePath = filepath.Join(ConfigDirPath, DefaultAPISpecFileName)
```

Comment:
Do we need this commented code?

Reply:
I dnt think so aiya. But i didnt comment this line aiya.